manageiq-operator/pkg/helpers/miq-components/httpd_conf.go

```diff
@@ -65,6 +62,9 @@ Options SymLinksIfOwnerMatch
   RewriteRule .* ws://remote-console:3000%%{REQUEST_URI}  [P,QSA,L]
   ProxyPassReverse /ws/console ws://remote-console:3000/ws/console
 
+  RewriteRule ^/ http://ui:3000%%{REQUEST_URI} [P,QSA,L]
```

Comment:
I think this will now try to serve everything including SAML, OIDC, etc. breaking external authentication.

Reply:
I will test with oidc also. I will let you known.